packages/common/src/models/gradtrak.ts

```diff
@@ -0,0 +1,202 @@
+import mongoose, { Document, InferSchemaType, Schema } from "mongoose";
+
+export const GradtrakCustomEventSchema = new Schema({
+  start_time: {
+    type: String,
+    required: true,
+    trim: true,
+    alias: "start",
+  },
+  end_time: {
+    type: String,
+    required: true,
+    trim: true,
+    alias: "end",
+  },
+  title: {
+    type: String,
+    required: false,
+    alias: "name",
+  },
+  location: {
```

Comment:
I would assume some of this additional information (`location`, `start_time`, `end_time`, `days_of_week`) probably isn't required for GradTrak because it doesn't deal with data of that specificity.